clang/lib/CIR/CodeGen/CIRGenModule.cpp

```diff
@@ -944,6 +944,19 @@ CIRGenModule::getOrCreateCIRGlobal(StringRef MangledName, mlir::Type Ty,
         GV.setSectionAttr(builder.getStringAttr(SA->getName()));
     }
 
+    const VisibilityAttr *VA = D->getAttr<VisibilityAttr>();
+    CIR_VisibilityAttr cir_visibility;
+    if (VA) {
```

Comment:
You can also rewrite this as:
```
 cir_visibility = CIR_VisibilityAttr::get(
          builder.getContext(),
          VA ? getCIRVisibilityKindFromClangVisibility(VA->getVisibility()) : VisibilityKind::Default);
```
Alternatively you can change the builder to have a default argument for the visibility kind.